kotlin/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'com.android.application' version '8.1.2' apply false
     id 'com.android.library' version '8.1.2' apply false
-    id 'org.jetbrains.kotlin.android' version '1.9.10' apply false
+    id 'org.jetbrains.kotlin.android' version '2.0.0' apply false
```

Comment:
TODO: Revert changes ? (need to ensure there is nothing needed in bumping this version).

@todd-spruceid brought up a good point that we should try to maintain legacy versions (when we don't need API updates, etc.) so that we can cover the broadest (older) phone versions in the population of users.

Reply:
Does bumping this plugin version actually impact end users? My understanding is that this is just the Kotlin version, which shouldn't impact the compiled application.